service/awsconfig/v8/key/key_test.go

```diff
@@ -156,10 +156,13 @@ func Test_ClusterOrganization(t *testing.T) {
 }
 
 func Test_ClusterTags(t *testing.T) {
+	installName := "test-install"
+
 	expectedID := "test-cluster"
 	expectedTags := map[string]string{
 		"kubernetes.io/cluster/test-cluster": "owned",
 		"giantswarm.io/cluster":              "test-cluster",
+		"giantswarm.io/installation":         "test-install",
```

Comment:
WDYT about naming it `giantswarm.io/host-cluster` to be more Giant Swarm agnostic and more usable outside. Also we use the term in several places already.

Reply:
I'm OK with it as long as we use the same tag in guest and host clusters.

Reply:
I don't see it being a problem. Do you?

Reply:
No should be fine let's change it.

Reply:
Our wording in opsctl is installation. Not sure what the reason for @marians was to do so.

Reply:
@kopiczko I'm staying with installations as requested by Timo which aligns with the installations repo and opsctl.

See https://github.com/giantswarm/giantswarm/issues/2775#issuecomment-371757090

Reply:
opsctl is 100% internal tool. It's fine to use whatever term there.

Reply:
Let's discuss in #2775.